app/controllers/your-psp/get.controller.js

```diff
@@ -3,7 +3,7 @@
 const { response } = require('../../utils/response')
 const { getCredentialByExternalId, getCurrentCredential, getSwitchingCredentialIfExists, hasSwitchedProvider } = require('../../utils/credentials')
 const { getTaskList, isComplete } = require('./kyc-tasks.service')
-const enableStripeOnboardingTaskList = process.env.ENABLE_STRIPE_ONBOARDING_TASK_LIST
+const enableStripeOnboardingTaskList = process.env.ENABLE_STRIPE_ONBOARDING_TASK_LIST === 'true'
```

Comment:
Can we remove the `=== 'true'`?
Change it to:
const enableStripeOnboardingTaskList = process.env.ENABLE_STRIPE_ONBOARDING_TASK_LIST || ''